src/components/header/index.jsx

```diff
@@ -1,14 +1,27 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
 import Container from '../container';
 import logo from '../../assets/logo_s.png';
-import s from './index.module.css';
+import s from './index.module.scss';
 
 
-const MENU = ['Menu 1', 'Menu 2', 'Menu 3', 'Menu 4'];
+const MENU = ["Menu 1", "Menu 2", "Menu 3", "Menu 4"];
+
 
 const Header = () => {
+  const [small, setSmall] = useState(false);
+
+  useEffect(() => {
+    const scrollFunc = () => window.pageYOffset >= 60 ? setSmall(true) : setSmall(false);
```

Comment:
Это можно написать короче :)
```suggestion
    const scrollFunc = () => setSmall(window.pageYOffset >= 60);
```